### DISCUSSION TITLE
What is the best way to update dependencies like Parse-Dashboard?

### DISCUSSION THREAD

**Discussion by jaysonng** [netreconlab/parse-server-swift#60] [Q&A] [answered]
So currently, Parse-Dashboard is at version 5.3.

The version in this repo still uses 5.2, what is the best way to update and use 5.3 seeing as this pulls from your other repos? (because there's a bug in the dashboard and I want to see if 5.3 fixes it)

**Comment by cbaker6** [comment]
The docker-compose file in this repo is using my custom images of parse. These are only intended to be examples. `parse-server-swift` can work with any parse-server or parse-dashboard as long as you configure correctly. An example of using the standard parse-dashboard image can be found here: https://github.com/netreconlab/parse-hipaa/blob/ea87809cfc5d74a574d1729d14a336f36718cbcd/docker-compose.no.hipaa.yml#L45-L57.

**Reply by jaysonng** [reply]
Thanks. Ive been looking into that and seems like a docker question now, but ill ask anyway - so It seems like i need a repo thats registered under docker hub or github package correct?

Basically I want to pull in the parse-swift-server version i have on my github repo (which is a fork of this your parse-swift-server). 

Then also the custom parse-server i already have setup on my repo as well. So thats where im at. Looking into github packages and docker hub.

**Reply by cbaker6** [reply, marked as answer]
If you change the line below: https://github.com/netreconlab/parse-server-swift/blob/79a39f037055ff14deffb0c7d83dff712575b237/docker-compose.yml#L77

to `netreconlab/parse-hipaa:6.4.0-dashboard` and you will have the latest released Parse Server and Dashboard.

**Reply by cbaker6** [reply]
> so It seems like i need a repo thats registered under docker hub or github package correct?

Yes, you can use GitHub actions to push your repo on new releases, similar to my repos (e.g. [parse-hipaa](https://github.com/netreconlab/parse-hipaa/blob/016ed4ad4f5d504c4a88eb05cc26b2992b3b85da/.github/workflows/release.yml#L23-L80) which pushes to [docker hub](https://hub.docker.com/r/netreconlab/parse-hipaa). Docker has server examples online or you can look at the parse-server repo
